Dockerfile

```diff
@@ -2,13 +2,17 @@ FROM  rustlang/rust:nightly AS builder
 WORKDIR /cennznet
 COPY . /cennznet
 
-ENV RUST_VERSION 1.44.1
+ARG RUST_VERSION=1.44.1
```

Comment:
By the way, 1.44.1 is the default version when no argument is passed to the docker. The actual version is coming from the Rust CircileCI context.